### PR TITLE
circle.yml: initial commit.

### PR DESCRIPTION
**Pull request by barakwei**

### CODE REVIEW

**Comment by StatusReport** [comment]
For some reason the configuration to build forked pull requests was not enabled. Can you push again?

**Comment by barakwei** [comment]
How did you notice that so fast? WTF? Anyway thanks - looks good and I'm merging. It's a shame their tests surround only the core tweak object and not anything else.

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        CODE_SIGNING_REQUIRED=NO
+        CODE_SIGN_IDENTITY=
+        PROVISIONING_PROFILE=
+        -destination 'platform=iOS Simulator,name=iPhone 7,OS=latest'
+        -project FBTweak.xcodeproj
+        -scheme "FBTweak"
+        clean build test |
+      tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+      xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
+
```
Comment (StatusReport) [inline circle.yml:18]: Newline :)

**Comment by barakwei** [comment]
well shit. next time.

**Comment by barakwei** [comment]
Please set master branch to protected with circleCI.

**Comment by StatusReport** [comment]
Done.

**Pull request merged**